frontend/src/metabase/visualizations/visualizations/Scalar.jsx

```diff
@@ -206,7 +206,8 @@ export default class Scalar extends Component {
     // use the compact version of formatting if the component is narrower than
     // the cutoff and the formatted value is longer than the cutoff
     const displayCompact =
```

Comment:
Should we add a special guard for `null`? Since `fulllScalarValue` (and also `compactScalarValue`) is either `null` or a string, the whole conditional with `displayCompact` is kind of redundant.

Reply:
Yeah good call. It's better to be explicit.

I considered updating our formatting code to always return a string ("" for null), but that feels like a more dangerous change.